src/mscorlib/src/System/Globalization/CultureInfo.cs

```diff
@@ -26,25 +26,29 @@
 //
 ////////////////////////////////////////////////////////////////////////////
 
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Diagnostics.Contracts;
+using System.Runtime.Serialization;
+using System.Threading;
+#if FEATURE_APPX && !FEATURE_COREFX_GLOBALIZATION
```

Comment:
What does `FEATURE_COREFX_GLOBALIZATION` mean now that you have gotten rid of it? Should this ifdef be deleted?